src/assets/stylesheets/index.scss

```diff
@@ -1170,6 +1170,37 @@ img.cookie-img {
   }
 }
 
+/* search-suggest
+=================================================================== */
+
+.search-box-suggest {
```

Comment:
Can you move this part to https://github.com/Scrivito/scrivito_example_app_js/blob/master/src/assets/stylesheets/_navigation.scss#L336, since we want to make the css more modular?

Reply:
You can cut it out and place it in a separate SCSS file. But this does not mean that you can just copy an paste it .in a different project. It depends to this site design. For me it does not make sense.